boards/apollo3/redboard_artemis_nano/Makefile

```diff
@@ -49,4 +47,4 @@ test:
 	mkdir -p $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/deps/
 	$(Q)cp layout.ld $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/deps/
 	$(Q)cp ../../kernel_layout.ld $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/
-	$(Q)RUSTFLAGS="$(RUSTC_FLAGS_TOCK)" OBJCOPY=${OBJCOPY} PORT=$(PORT) $(CARGO) test $(CARGO_FLAGS_TOCK_NO_BUILD_STD) $(NO_RUN) --bin $(PLATFORM) --release
+	$(Q)OBJCOPY=${OBJCOPY} PORT=$(PORT) $(CARGO) test $(NO_RUN) --bin $(PLATFORM) --release
```

Comment:
Do we know why all the `apollo3` boards had this extra use of `RUSTFLAGS` for `OBJCOPY`? They're the only ones... an old copy-paste artifact, or doing something meaningful?

Asked another way: has another with apollo3 hardware tested this?

Reply:
That's a cargo test command.

Reply:
well, that's what I get for not expanding the context on github